components/ingest-service/backend/elastic/mappings/node_state.go

```diff
@@ -121,6 +129,7 @@ var nodeProps = `
 	},
 	"resource_names": {
 		"type": "keyword",
+		"normalizer": "case_insensitive",
```

Comment:
The `case_insensitive` normalizer is needed to allow wildcard filters to be case insensitive.